wifiphisher/common/phishingpage.py

```diff
@@ -142,202 +106,74 @@ def update_payload_path(self, filename):
-            copyfile(path, self.get_path_static() + filename)
-            self._extra_files.append(self.get_path_static() + filename)
+            copyfile(path, self.static_path + filename)
+            self._extra_files.append(self.static_path + filename)
             return filename
 
     def remove_extra_files(self):
-        """
-        Removes extra used files (if any)
-
-        :param self: A PhishingTemplate object
-        :type self: PhishingTemplate
-        :return: None
-        :rtype: None
-        """
-
+        # type: () -> None
+        """Remove extra used files."""
         for filename in self._extra_files:
             if os.path.isfile(filename):
                 os.remove(filename)
 
     def __str__(self):
-        """
-        Return a string representation of the template.
-
-        :param self: A PhishingTemplate object
-        :type self: PhishingTemplate
-        :return: the name followed by the description of the template
-        :rtype: str
-        """
-
-        return self._display_name + "\n\t" + self._description + "\n"
+        # type: () -> str
+        """Return a string representation of the template."""
+        return "{display_name}\n\t{_description}\n".format(
+            display_name=self.display_name, _description=self._description)
 
 
 class TemplateManager(object):
-    """ This class handles all the template management operations """
+    """Handles all the template management operations."""
 
     def __init__(self):
-        """
-        Construct object.
-
-        :param self: A TemplateManager object
-        :type self: TemplateManager
-        :return: None
-        :rtype: None
-        """
-
+        """Initialize the class."""
         # setup the templates
         self._template_directory = constants.PHISHING_PAGES_DIR
 
         page_dirs = os.listdir(constants.PHISHING_PAGES_DIR)
 
-        self._templates = {}
+        self.templates = {}
 
         for page in page_dirs:
             if os.path.isdir(page) and self.is_valid_template(page)[0]:
-                self._templates[page] = PhishingTemplate(page)
+                self.templates[page] = PhishingTemplate(page)
 
         # add all the user templates to the database
         self.add_user_templates()
 
-    def get_templates(self):
-        """
-        Return all the available templates.
-
-        :param self: A TemplateManager object
-        :type self: TemplateManager
-        :return: all the available templates
-        :rtype: dict
-        """
-
-        return self._templates
-
     def is_valid_template(self, name):
-        """
-        Validate the template
-        :param self: A TemplateManager object
-        :param name: A directory name
-        :type self: A TemplateManager object
-        :return: tuple of is_valid and output string
-        :rtype: tuple
-        """
-
+        # type: (str) -> Tuple[bool, str]
+        """Validate the template."""
```

Comment:
Validate the template.

Looks that a `config.ini` file and an `html` directory are placed.